README.md

```diff
@@ -6,7 +6,7 @@ RPCS3
 
 The world's first open-source PlayStation 3 emulator/debugger written in C++ for Windows and Linux.
 
-You can find some basic information in our [**website**](https://rpcs3.net/).
+You can find some basic information in our [**website**](https://rpcs3.net/). Game info is being moved to the [**wiki**](https://wiki.rpcs3.net/).
```

Comment:
What's "Game info is being moved to the [**wiki**](https://wiki.rpcs3.net/)"? Doesn't make sense. Edit next line and make it "visit our forums, wiki and our discord server".

Reply:
I think the text is best served if each location has a description of its function.

Reply:
It doesn't make sense because anything stuff is not being **moved**, wiki is being populated with mostly new data and some stuff from the existing forum threads.

Reply:
Alright that works